protocol/synthetix/subgraph/codegen.sh

```diff
@@ -38,9 +38,10 @@ codegen base-mainnet-andromeda 8453 "synthetix-omnibus:latest@andromeda"
 
 # Mainnet EVM & OP
 codegen mainnet 1 "synthetix-omnibus:latest@main"
+codegen sepolia 11155111 "synthetix-omnibus:latest@main"
+
 codegen optimism-mainnet 10 "synthetix-omnibus:latest@main"
 
 # Arbitrum
 codegen arbitrum-mainnet 42161 "synthetix-omnibus:latest@arbthetix"
-# Note currently borked
-# codegen arbitrum-sepolia 421614 "synthetix-omnibus:latest@arbthetix"
+codegen arbitrum-sepolia 421614 "synthetix-omnibus:latest@arbthetix"
```

Comment:
Please do not use arbthetix. `main` is the way
```suggestion
codegen arbitrum-sepolia 421614 "synthetix-omnibus:latest@main"
```